lib/elixir/lib/task.ex

```diff
@@ -133,7 +131,7 @@ defmodule Task do
   works with anonymous functions. That's because anonymous functions expect
   the same module version to exist on all involved nodes. Check the `Agent` module
   documentation for more information on distributed processes as the limitations
-  described in the agents documentation apply to the whole ecosystem.
+  described there apply to the whole ecosystem.
```

Comment:
actually looking again now, I don't know if it's even right or very necessary, but it would better go after distributed processes.

"Check the `Agent` module documentation for more information on distributed processes, as the limitations described there apply to the whole ecosystem."